src/test/java/com/epam/tc/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.Test;
+
+public class CalculatorDivideTest {
+    Calculator calculator = new Calculator();
+
+    @Test(dataProvider = "")
+    public void divideTest() {
+        double actualResult = calculator.div(2,2);
+        assertThat(actualResult)
+                .as("Dividing result")
+                .isEqualTo(1);
+    }
```

Comment:
1. One test set is not enough, please add more (\+ for all test classes) 
2. Please pay attention how to use DataProviders: https://testng.org/doc/documentation-main.html#parameters-dataproviders
3. Move DataProvider to separate class

Reply:
done